mp/src/game/shared/momentum/mom_rocket.cpp

```diff
@@ -171,6 +222,45 @@ void CMomRocket::DestroyTrail()
     }
 }
 
+void CMomRocket::CreateRocketExplosionEffect(trace_t *pTrace, CBaseEntity *pOther)
+{
+    static ConVarRef mom_rj_sounds("mom_rj_sounds");
+    int iEntIndex = pOther->entindex();
+    CWeaponID m_hWeaponID = WEAPON_ROCKETLAUNCHER;
+    Vector vecOrigin = GetAbsOrigin();
+    CBaseEntity *pOwner = GetOwnerEntity();
+    float flDamage = GetDamage();
+    float flRadius = GetRadius();
+    CPVSFilter filter(vecOrigin);
+
+    switch (mom_rj_particles.GetInt())
+    {
+    case 0:
+        TE_TFExplosion(filter, 0.0f, vecOrigin, pTrace->plane.normal, m_hWeaponID, iEntIndex);
```

Comment:
If particles are turned off there's still an explosion?

Reply:
When `mom_rj_particles` is 0, `TE_TFExplosion` doesn't produce any particles but is still responsible for producing the explosion sound depending on the value of `mom_rj_sounds` which I imagine people would still want even if they disable particles (since otherwise they could just set the sound cvar to 0 as well).

Reply:
Then move the `case 0:` label down to right above the `case 2:` one since they're the same:

```
case 0:
case 2:
   TE_TFExplosion(filter ... );
   break;
```